install-solc.sh

```diff
@@ -103,7 +103,7 @@ initOS
 
 # assemble expected release artifact name
 if [ "$OS" = "darwin" ]; then
-    sudo brew install solidity@4
+    sudo brew install solidity@5
```

Comment:
Need `brew tap ethereum/ethereum`? or not?

Reply:
Yes, modified.